tskv/src/mem_cache/series_data.rs

```diff
@@ -1,5 +1,5 @@
 use std::collections::{HashMap, LinkedList};
-use std::ops::Bound::Included;
+// use std::ops::Bound::Included;
```

Comment:
remove unused code

Reply:
ok，Should I submit the PR again here or merge it with my new code?